glslc/README.asciidoc

```diff
@@ -390,10 +387,17 @@ output.
 The `GL_GOOGLE_cpp_style_line_directive` extension is implicitly turned on by
 the `GL_GOOGLE_include_directive` extension.
 
-It is invalid to have any '\' or '/' characters in any filename provided to
-an `#include` directive. The presence of any such characters may lead to
-unexpected behavior on different systems.
+The file argument to `#include` must be enclosed in double quotes.  It must be a
+relative path, using whatever path separator the OS supports.  However, the last
+path element -- the name of the file itself -- must not contain either '/' or
+'\', regardless of which path separator is used.  For example, the following is
```

Comment:
I think this is a weird way to explain it. 
We should say that for such an example, we'll use 'ame.vert' as a filename in path 'dir\subdir\subsubsdir\filen'.   And whether that path is valid depends on the current platform.